The NASA performance test is skipped becuase it is very sentitive to what filesystem is used, causing timeouts.
Author: micketeer@gmail.com
--- nc_test4/run_par_test.sh.in.orig	2022-08-17 11:14:21.827283219 +0200
+++ nc_test4/run_par_test.sh.in	2022-08-17 11:12:49.547459608 +0200
@@ -37,9 +37,9 @@
 echo "Testing collective writes with some 0 element writes..."
 @MPIEXEC@ -n 4 ./tst_parallel5
 
-echo
-echo "Parallel Performance Test for NASA"
-@MPIEXEC@ -n 4 ./tst_nc4perf
+#echo
+#echo "Parallel Performance Test for NASA"
+#@MPIEXEC@ -n 4 ./tst_nc4perf
 
 echo
 echo "Parallel I/O test for Collective I/O, contributed by HDF Group."
